pkg/util/lifted/discovery.go

```diff
@@ -39,7 +41,10 @@ func GetDeletableResources(discoveryClient discovery.ServerResourcesInterface) m
 	preferredResources, err := discoveryClient.ServerPreferredResources()
 	if err != nil {
 		if discovery.IsGroupDiscoveryFailedError(err) {
-			klog.Warningf("Failed to discover some groups: %v", err.(*discovery.ErrGroupDiscoveryFailed).Groups)
+			var tErr *discovery.ErrGroupDiscoveryFailed
```

Comment:
This is the `lifted` code, I'm hesitant to change it unless there is a good reason.
Can we contribute to the upstream first?

Reply:
Thanks for pointing out the question of whether the lifted code can be interpreted to mean that it will never be modified

Reply:
The lifted code can be touched if there is a need(like fixing a bug). 
For the errorlint check, I'm not sure if worth to do it according to the benefits we can get from this.